semantics/src/traversal/functions.rs

```diff
@@ -28,8 +28,6 @@ use fe_parser::ast as fe;
 use fe_parser::span::Spanned;
 use std::rc::Rc;
 
-/// Gather context information for a function definition and check for type
```

Comment:
We should keep doc comments on all public functions

Reply:
Good point. Removal was by accident.